.github/ISSUE_TEMPLATE/something_else_open_issue.yml

```
@@ -1,13 +1,12 @@
-name: Open empty issue
+name: Something else? Open empty an issue
```

"Open an empty issue"

---

```suggestion
name: Open empty an issue
```

I renamed it to be the last one in the list, but in fact the order is based on the filename, so it's OK

---

Oups my bad I misread your correction